docs/user-guide/how-to.md

```diff
@@ -186,6 +187,15 @@ permissions. To understand why an object is not being propagated to a namespace,
 use `kubectl hns describe <ns>`, where `<ns>` is either the source (ancestor) or
 destination (descendant) namespace.
 
+<a name="use-hrq"/>
+
+### Limit Resources over parent namespaces
+
+HNC has an object called HirarchicalResourceQuota that is similar to ResourceQuota
+but across all namespace tree. It allows you to distribute tour resources between
```

Comment:
"across all namespace tree." -> "across all the namespaces in a hierarchy."

"tour resources" -> "your resources"